### ISSUE TITLE
HA Watchdog does not start

### ISSUE THREAD

**Issue by roopesh** [TD22057/insteon-mqtt#421]
Not sure what else to include except this.  I see this in the HA Supervisor logs:
```
21-05-23 17:15:09 INFO (SyncWorker_2) [supervisor.docker.addon] Starting Docker add-on td22057/amd64-insteon-mqtt with version 0.9.2
21-05-23 17:15:56 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-1158314' coro=<Scheduler._run_task.<locals>._wrap_task() done, defined at /usr/src/supervisor/supervisor/misc/scheduler.py:58> exception=TypeError('an integer is required (got type str)')>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/misc/scheduler.py", line 62, in _wrap_task
    await task.coro_callback()
  File "/usr/src/supervisor/supervisor/misc/tasks.py", line 416, in _watchdog_addon_application
    if addon.in_progress or await addon.watchdog_application():
  File "/usr/src/supervisor/supervisor/addons/addon.py", line 479, in watchdog_application
    return await self.sys_run_in_executor(check_port, self.ip_address, port)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/supervisor/supervisor/utils/__init__.py", line 43, in check_port
    result = sock.connect_ex((str(address), port))
TypeError: an integer is required (got type str)
```

**Comment by krkeegan** [comment]
No idea how watchdog works.  Everything should work if you disable that in supervisor.

**Comment by krkeegan** [comment]
This looks to be caused by an error in HomeAssistant see this merge https://github.com/home-assistant/supervisor/pull/2903

The fix has been merged as part of Supervisor version 2021.05.1.  I am closing this issue on the assumption that the Supervisor update will fix the issue.  If it still remains after the update please reopen.
